src/Native/Runtime/GCMemoryHelpers.inl

```diff
@@ -7,14 +7,21 @@
 // Unmanaged GC memory helpers
 //
 
+#ifndef DACCESS_COMPILE
```

Comment:
Is this correct? The GPTR_DECL is inside of this ifndef and so it seems the g_lowest_address and g_highest_address could have stayed plain uint8*. The GPTR_DECL for non-dac build is defined as
```C
#define GPTR_DECL(type, var) \
    extern type* var
```